payload_relay.ino

```diff
@@ -0,0 +1,40 @@
+#if false
+
+#ifndef PAYLOAD_THRESHOLD
+#define PAYLOAD_THRESHOLD 14.5 // height of one floor in Baldwin staircase in units of feet
+#endif
+
+#ifndef COUNT_THRESHOLD
+#define COUNT_THRESHOLD 200
+#endif
+
+#include "Altimeter.h"
+
+Altimeter m_altimeter;
+
+int numAltimeterReadings = 0;
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial) { /* Wait for Serial monitor to connect */ }
+  Serial.println("Connecting to sensor...");
+  m_altimeter.initialize();
+}
+
+void loop() {
```

Comment:
This may be a me thing, but I don't see where we're limiting/controlling the rate at which we're taking readings? We will want to do this and set it to either 200 or 100 Hz.

Reply:
I think I accounted for this when I said `if (altitudeOutput < PAYLOAD_THRESHOLD)` but I could be wrong. I would need Riley to weigh in.

Reply:
The altimeter itself can only give readings at a rate of 200Hz. The limitation is in the getAltitude call itself.

Reply:
How would I go about rate limiting the `getAltitude()` call?

Reply:
What I'm saying is you don't need to. If you call it too quickly, it waits for a new reading from the altimeter before returning.

Reply:
oh okay, i gotcha. then yeah, don't worry about manually limiting the data rate